src/petTrack.js

```diff
@@ -0,0 +1,109 @@
+// PROTOTYPE SYNTAX PET
+
+const MAXIMUM_FITNESS = 10;
+const MINIMUM_FITNESS = 0; 
+const FITNESS_INCREASE = 4;
+const MINIMUM_HUNGER = 0;
+const REDUCE_HUNGER = 3;
+const ACCEPTABLE_HUNGER = 5
+const ACCEPTABLE_FITNESS = 3
+const MAXIMUM_HUNGER = 10
+const MAXIMUM_AGE = 30
+
+function Pet(name) {
+    this.name = name;
+    this.age = 0;
+    this.hunger = MINIMUM_HUNGER;
+    this.fitness = MAXIMUM_FITNESS;
+    this.children = [];
+}
+
+Pet.prototype = {
+    get isAlive() {
+        return this.fitness > MINIMUM_FITNESS && this.hunger < MAXIMUM_HUNGER && this.age < MAXIMUM_AGE;
+    }
+}
+
+Pet.prototype.growUp = function () {
+    
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive');
+      }  
+    
+    this.age += 1;
+    this.hunger += 5;
+    this.fitness -= 3;
+
+    if (this.fitness < 0) {
+    this.fitness = MINIMUM_FITNESS;
+    }
+}
+
+Pet.prototype.walk = function () {
+
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive');
+      }  
+   
+    if ((this.fitness + FITNESS_INCREASE) > MAXIMUM_FITNESS) {
+        this.fitness = MAXIMUM_FITNESS;
+    } else {
+        this.fitness += FITNESS_INCREASE;
+    }
+}
+
+Pet.prototype.feed = function () {
+
+    if (!this.isAlive) {
+        throw new Error('Your pet is no longer alive');
+      }    
+
+    if ((this.hunger - REDUCE_HUNGER) >= 0) {
+        this.hunger -= REDUCE_HUNGER;
+    } else {
+        this.hunger = MINIMUM_HUNGER
+    }
+}
+
+Pet.prototype.checkUp = function () {
+    if (!this.isAlive) {
+        return 'Your pet is no longer alive'
+    }
+    
+        if (this.fitness <= ACCEPTABLE_FITNESS && this.hunger >= ACCEPTABLE_HUNGER) {
+    return 'I am hungry AND I need a walk'
+    }
+    if (this.fitness <= ACCEPTABLE_FITNESS) {
+    return 'I need a walk'
+    }
+    if (this.hunger >= ACCEPTABLE_HUNGER) {
+    return "I am hungry"
+    } else {
+    return 'I feel great'
+    }
+}
+
```

Comment:
The formatting is pretty off kilter here. Remember you should indent your code by one layer when moving further into a function, and return back when exiting. You can also condense the code a bit by returning each if statement on its own line, and add a default condition to return should none of the earlier if statements be satisfied. As an example this function could look like this:
```
Pet.prototype.checkUp = function () {
    if (!this.isAlive) return "Your pet is no longer alive";
    if (this.fitness <= ACCEPTABLE_FITNESS) return "I need a walk";
    if (this.hunger >= ACCEPTABLE_HUNGER) return "I am hungry";
    if (
        this.fitness <= ACCEPTABLE_FITNESS &&
        this.hunger >= ACCEPTABLE_HUNGER
    ) {
        return "I am hungry AND I need a walk";
    }
    
    return "I feel great";
};
```